### PR TITLE
Build SBI case study page

### LINKED ISSUE #6175: [SBI (English)] - Build engage page
## Requirements

Please build a takeover and an engage page based on this [brief](https://docs.google.com/spreadsheets/u/1/d/158eh4AHxfgssQLu8pfCp0pKFZduwE4BpJ8y0-rfAs6k/edit?usp=sharing) and [design](1955):
 
- [x] build engage page
- [x] add engage page to `/engage/index.html`
- [ ] test the engage page on [https://socialdebug.com/](https://socialdebug.com/) or [http://debug.iframely.com/](http://debug.iframely.com/) and [https://cards-dev.twitter.com/validator](https://cards-dev.twitter.com/validator)

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Build SBI case study engage page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage/sbi-casestudy
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that it matches the [copy doc](https://docs.google.com/document/d/1ky7FmqiSEBi_fDMo3d2p7oeIrFhvSCCCJmohd6rc6uQ)

## Issue / Card

Fixes #6175

### CODE REVIEW

**Comment by anasereijo** [comment]
can you add the logo as per the japanese version https://jp.ubuntu.com/engage/sbi?utm_source=takeover&utm_campaign=CY19_DC_Japan_CStudy_SBIJP

**Comment by steverydz** [comment]
> can you add the logo as per the japanese version https://jp.ubuntu.com/engage/sbi?utm_source=takeover&utm_campaign=CY19_DC_Japan_CStudy_SBIJP

@anasereijo done https://ubuntu-com-canonical-web-and-design-pr-6234.run.demo.haus/engage/sbi-casestudy

File: templates/engage/sbi-casestudy.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+<blockquote class="p-pull-quote">
+  <p class="p-pull-quote__quote">There was no risk of lock-in with Canonical’s OpenStack. It’s close to the upstream version, so we have the freedom to fully support internally, if we want to.</p>
+  <cite class="p-pull-quote__citation">Georgi Georgiev, CIO, SBI BITS</cite>
+</blockquote>
+
+Taking just a few weeks to implement, SBI BITS are benefitting from an OpenStack cloud that is cost effective from both CAPEX and OPEX perspectives and providing them the confidence to deploy across the entire SBI Group
+
+In this case study, learn how SBI BITS: 
+- Met regulatory compliance through Canonical&rsquo;s OpenStack across two sites, utilising CEPH storage, and deployed using MaaS and Juju.
```
Comment (anthonydillon) [inline templates/engage/sbi-casestudy.md]: It's MAAS not MaaS.

File: templates/engage/sbi-casestudy.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+     header_lang: en
+     webinar_code:
+     form_include: en
+     form_id: 3422
+     form_return_url: "https://pages.ubuntu.com/CStudy_SBI-TY.html"
+---
+
+SBI BITS provides IT services and infrastructure to the SBI Group &mdash; Japan&rsquo;s market leading financial services company group &mdash; which is made up of over 250 companies, and 6,000 employees.
+
+To increase time to market and meet heavy client requirements, SBI BITs was looking for alternative solutions beyond bare metal servers and decided upon OpenStack. After evaluating their existing suppliers, SBI BITS turned to Canonical to bring in the external support and expertise required to move into production with an economical, flexible solution.
```
Comment (anthonydillon) [inline templates/engage/sbi-casestudy.md]: Increasing time to market is a bad thing?

File: templates/engage/sbi-casestudy.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+<blockquote class="p-pull-quote">
+  <p class="p-pull-quote__quote">There was no risk of lock-in with Canonical’s OpenStack. It’s close to the upstream version, so we have the freedom to fully support internally, if we want to.</p>
+  <cite class="p-pull-quote__citation">Georgi Georgiev, CIO, SBI BITS</cite>
+</blockquote>
+
+Taking just a few weeks to implement, SBI BITS are benefitting from an OpenStack cloud that is cost effective from both CAPEX and OPEX perspectives and providing them the confidence to deploy across the entire SBI Group
```
Comment (anthonydillon) [inline templates/engage/sbi-casestudy.md]: them **with** the confidence

Comment (anthonydillon) [reply templates/engage/sbi-casestudy.md]: s/benefitting/benefiting

Comment (anthonydillon) [reply templates/engage/sbi-casestudy.md]: Missing full stop.

**Pull request merged**